Sources/SwiftGRPCNIO/GRPCError.swift

```diff
@@ -143,7 +146,10 @@ extension GRPCServerError: GRPCStatusTransformable {
     case .responseProtoSerializationFailure:
       return GRPCStatus(code: .internalError, message: "could not serialize response proto")
 
-    case .requestCardinalityViolation:
+    case .requestCardinalityViolationTooFew:
```

Comment:
nit: these names don't sound quite right now, `too{Few,Many}Requests`?

Reply:
Done, thanks!

Reply:
Ah, I meant just replacing the names!
- `requestCardinalityViolationTooFew` -> `tooFewRequests`
- `requestCardinalityViolationTooMany` -> `tooManyRequests`

Reply:
Renamed again :-)